gapis/api/gles/api/draw_commands.api

```diff
@@ -374,6 +374,31 @@ sub void WriteFramebufferAttachment(FramebufferAttachment a) {
   }
 }
 
+// DependencyReadTexture read all relevant fields of texture to create dependencies
+sub void DependencyReadTexture(ref!Context ctx, ref!Texture texture) {
+  read(texture.Image.Data)
+
+  // Sampling may be affected by all these parameters
+  _ = texture.SwizzleR
+  _ = texture.SwizzleG
+  _ = texture.SwizzleB
+  _ = texture.SwizzleA
+  _ = texture.BorderColor
+  _ = texture.BorderColorI
+  _ = texture.MinFilter
+  _ = texture.MagFilter
+  _ = texture.WrapS
+  _ = texture.WrapT
+  _ = texture.WrapR
+  _ = texture.MinLod
+  _ = texture.MaxLod
+  _ = texture.BaseLevel
+  _ = texture.MaxLevel
+  _ = texture.DepthStencilTextureMode
+  _ = texture.CompareMode
+  _ = texture.CompareFunc
+}
```

Comment:
missing are `MaxAnisotropy` and `DecodeSRGB`

Reply:
thanks! done